usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateRouteColorAndTextContrast.java

```diff
@@ -26,18 +27,21 @@
 /**
  * Use case to validate that a Route color contrast with a Route text color.
  */
-public class ValidateRouteColorAndTextContrast {
+public class ValidateRouteColorAndTextContrast implements ValidationUsecase {
```

Comment:
@maximearmstrong FYI, use cases that implement logic related to specific GTFS file now implement an interface in order to manipulate them in Main method. 
The end goal is to be able to call the `.execute` method of each file specific use case inside of a loop.

We can discuss it later during one of our syncs.